app/uk/gov/hmrc/cbcr/repositories/DocRefIdRepository.scala

```diff
@@ -35,7 +37,9 @@ class DocRefIdRepository @Inject()(mongo: MongoComponent, records: ReactiveDocRe
       mongoComponent = mongo,
       collectionName = "DocRefId",
       domainFormat = DocRefId.format,
-      indexes = Seq()
+      indexes = Seq(
+        IndexModel(ascending("id"), IndexOptions().name("id").unique(true).background(true))
```

Comment:
Are we sure entries in this collection do have unique IDs?

Reply:
isn't the point of an id that it's unique?

Reply:
No. Index is just a smaller, easier to search collection. It doesn't have to be unique, though it certainly helps.

Reply:
when I've seen it done on other services they use unique with ids 
so I'd assumed it was pattern
is there a reason it wouldn't be?

Reply:
as we use it to query the db I would assume that id is unique as doing something like "delete" would chop out more than on entry

Reply:
1. What happens if you add an index to an existing collection and entries are not unique? I'm assuming data loss. You can try googling or running an experiment locally to see what happens with the data.
2. If it results in data loss, you can try verifying whether data is unique by running a mongo query to get all entries, count by number of entries and filter if count is greater than 1 and count again to hide user data. If the query results in 0 then it's safe to put unique there.

Some indexes are unique, because the application started with an index and the index ensured data was unique. But now that we already have a table without an index, we don't know whether entries within would be unique or not. We need to verify this somehow and ensure introducing an index doesn't result in data loss.